src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -464,7 +461,7 @@ private unsafe static X11Window CreateGLXWindow(IntPtr display, int screen, Size
 			(int)visual->depth,
 			/* InputOutput */ 1,
 			visual->visual,
-			(UIntPtr)(XCreateWindowFlags.CWBackPixel | XCreateWindowFlags.CWColormap | XCreateWindowFlags.CWBorderPixel | XCreateWindowFlags.CWEventMask),
+			UIntPtr.Zero,
```

Comment:
Can you indicate why this needs to be zero and not the other many parameters that were present before?

Reply:
Let's also add a manual test that explains exactly what to do, we may forget to validate it in the future.